src/main/kotlin/watch/dependency/main.kt

```diff
@@ -98,6 +100,7 @@ private class AwaitCommand : DependencyWatchCommand(
 		.default(MavenCentralId)
 
 	private val coordinates by argument("COORDINATES", help = "Maven coordinates (e.g., 'com.example:example:1.0.0')")
+	private val terminal = Terminal()
```

Comment:
I wasn't able to inline this object inside `withProgressAnimation` due to this error:

```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x0000000149840164, pid=91544, tid=9219
#
# JRE version: OpenJDK Runtime Environment Zulu11.54+25-CA (11.0.14.1+1) (build 11.0.14.1+1-LTS)
# Java VM: OpenJDK 64-Bit Server VM Zulu11.54+25-CA (11.0.14.1+1-LTS, mixed mode, tiered, compressed oops, g1 gc, bsd-aarch64)
# Problematic frame:
# C  [jna3752758197612523520.tmp+0x10164]  ffi_prep_closure_loc+0x16bc
#
# No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /Users/saket/projects/dependency-watch/hs_err_pid91544.log
#
# If you would like to submit a bug report, please visit:
#   http://www.azul.com/support/
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
./run: line 3: 91544 Abort trap: 6           ./build/install/dependency-watch/bin/dependency-watch "$@"
```